CONTRIBUTORS.md

```diff
@@ -6,8 +6,14 @@ The following people have contributed to the implementation of xNVMe.
 Contributors to the current release
 -----------------------------------
 
+* Ankit Kumar, [Samsung]
+* Karl Bonde Torp, [Samsung]
+* Klaus Jensen, [Samsung]
+* LiadOz
```

Comment:
This is missing a space, it should be 
`Liad Oz`

Reply:
Thanks! Fixed.